CRM/Contribute/BAO/Contribution.php

```diff
@@ -3816,6 +3816,7 @@ public static function isSingleLineItem($id) {
    */
   public static function completeOrder($input, $recurringContributionID, $contributionID, $isPostPaymentCreate = FALSE) {
     if (!$contributionID) {
+      CRM_Core_Error::deprecatedFunctionWarning('v3api Contribution.repeattransaction. This handling will be removed around 5.70 (calling this function directly has never been supported outside core anyway)');
```

Comment:
@mattwire  it's the end of an era!!!

Reply:
@eileenmcnaughton Shouldn't this say "Contribution.completetransaction"?

Reply:
@mattwire nope - this is the calling of completeOrder to call repeattransaction that is deprected

Reply:
ok, the comment confuses me :-)